### PR TITLE
Extend doc about synchronization (gh-pages)

### PR DESCRIPTION
**Pull request by carlisia**
This is related to the feature improvement done here:

https://github.com/heptio/ark/pull/680

Signed-off-by: Carlisia <carlisia@grokkingtech.io>

### CODE REVIEW

File: master/about.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ Heptio Ark treats object storage as the source of truth. It continuously checks
 
 This allows restore functionality to work in a cluster migration scenario, where the original backup objects do not exist in the new cluster.
 
+Likewise, if a backup object exists in Kubernetes but not in the object storage, it will be deleted from Kubernetes since this means it is no longer in use.
```
Comment (nrb) [inline master/about.md]: I would probably word the first part `Likewise, if a backup object exists in Kubernetes but not in object storage, ...`.

**Comment by nrb** [comment]
One fairly minor comment, otherwise looks fine.

We'll also need a PR for master that adds this to the `docs/about.md` file in the `master` branch, so that docs for new versions include it.

File: master/about.md (hunk 1 above)
Comment (carlisia) [reply master/about.md]: Ok cool. I like the attention to detail!

File: master/about.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ Heptio Ark treats object storage as the source of truth. It continuously checks
 
 This allows restore functionality to work in a cluster migration scenario, where the original backup objects do not exist in the new cluster.
 
+Likewise, if a backup object exists in Kubernetes but not in object storage, it will be deleted from Kubernetes since this means it is no longer in use.
```
Comment (skriss) [inline master/about.md]: instead of "...this means it is no longer in use", I'd say something like "...the backup tarball no longer exists."

**Pull request merged**